docs/physics/plasma/plasma_plots/nebular_ionization_balance.py

```diff
@@ -1,7 +1,8 @@
 import os
 
 from matplotlib import colors
-from tardis import atomic, plasma_array, util
+from tardis import plasma_array, util
```

Comment:
Same as above, there is no `plasma_array`

Reply:
yep